README.md

```diff
@@ -1063,6 +1064,7 @@ vars:
 vars:
```

Comment:
Can you add this adjustment to the comment above? Currently it just says `# set your chained views threshold to 8 instead of 4`

Reply:
updated to have comments per variable to explain the reasoning